helm/aws-cloud-controller-manager-app/values.yaml

```diff
@@ -14,11 +14,11 @@ image:
 # can keep the ratio when scaling
 resources:
   limits:
-    cpu: 400m
-    memory: 600Mi
+    cpu: 100m
```

Comment:
I don't think we should reduce the limits. That can have strong effects, as @calvix pointed out (fewer CPU time slices, OOMKill if out of memory). Our problem at hand is allocation of pods to nodes, so changing requests should be enough?

Reply:
yeah, i think that's reasonable, or at least we can start with reducing the requests, that should be much safer